cmd/migration/README.md

```diff
@@ -0,0 +1,32 @@
+# Migration
+
+The migration service translates a daemon's datadir with a specific major version to the very next one.
+
+Currently supported versions:
+* from `v0` to `v1`, requires a daemon with exact verion `v0.9.1`
+
+## Build
+
+```sh
+$ make build-migration
+$ alias migration=./build/migration-<os>-<arch>
+```
+
+## Usage
+
+* Standalone binary version
+  ```sh
+  $ migration --password <wallet_password> [--source-version, --v0-datadir, --v1-datadir, --ocean-datadir, --no-backup]
+  ```
+* Docker version
+  ```sh
+  # Pull the latest image
+  $ docker pull ghcr.io/tdex-network/tdexd:latest
+  $ docker run --rm --volume path/to/datadir:/home/tdex/.tdex-daemon --entrypoint 'tdex-migration' ghcr.io/tdex-network/tdexd:latest --password <wallet_password> --/home/tdex/.tdex-daemon/oceand
```

Comment:
`--/home/tdex/.tdex-daemon/oceand` are you missing a flag name here?

Reply:
yes definetly